site/content/api/Tenant-API.md

```diff
@@ -270,7 +270,7 @@ The table below contains the properties which are used to configure a *Hono prot
 | *device-authentication-required*   | *no*      | *boolean*   | `true`        | If set to false, devices are not required to authenticate with the adapter before sending / receiving data. |
 
 Protocol Adapters SHOULD use the configuration properties set for a tenant when interacting with devices of that tenant, e.g. in order to make authorization decisions or to limit message rates per tenant etc.
-If for a certain protocol adapter no configuration is present it has the same semantic as a disabled protocol adapter with default values.
+If for a certain protocol adapter no configuration is present it has the same semantic as a configuration present with default values.
```

Comment:
this definition/clarification should be moved to the *Tenant Format* section FMPOV because it defines how the lack of config information needs to be handled at the tenant level.

Reply:
I moved it to the other section.